src/site/content/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -74,7 +76,9 @@ We worked around these legacy APIs by preventing content from entering the webpa
 
 With these mitigations in place, we reintroduced `SharedArrayBuffer` in Chrome 68 (July 2018), but only on desktop. The extra process requirements meant we couldn't do the same on mobile devices. It was also noted that Chrome's solution was incomplete, as we were only blocking 'incorrect' data formats, whereas it's possible (although unusual) that valid CSS/JS/images at guessable URLs can contain private data.
 
-Web standards folks got together to come up with a more complete solution. The solution was to give pages a way to say "I hereby relinquish my ability to bring other-origin content into this process without their opt-in". This declaration is done via [COOP and COEP headers](/coop-coep/) served with the page. The browser enforces that, and in exchange the page gains access to `SharedArrayBuffer` and other APIs with similar powers. Other origins can opt-in to content embedding via [`Cross-Origin-Resource-Policy`](<https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)>) or [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
+Web standards folks got together to come up with a more complete cross-browser solution. The solution was to give pages a way to say "I hereby relinquish my ability to bring other-origin content into this process without their opt-in". This declaration is done via [COOP and COEP headers](/coop-coep/) served with the page. The browser enforces that, and in exchange the page gains access to `SharedArrayBuffer` and other APIs with similar powers. Other origins can opt-in to content embedding via [`Cross-Origin-Resource-Policy`](<https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)>) or [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
+
+Firefox was the first to ship `SharedArrayBuffer` with this restriction, in version 79 (July 2020).
 
 Then, in December 2020, I wrote this article, and you read it. Hello.
```

Comment:
One more thing to add here. Starting M89 on Desktop we've the new flag RestrictSharedArrayBuffer to already gate SABs for test purposes. The behavior using this flag is the same as it will be in M91.

FYO starting in M89 a DevTools deprecation issue will be shown in case SABs are used in none gated context. Those warnings will point the users to this post. Do we need to add something to catch those users?

Reply:
Is that going to show up in chrome:flags?

Reply:
Yes, this new flag will show up in M89 (and is already in canary starting 89.0.4351.0)

Reply:
I can't find this flag in `chrome:flags` in Canary 89.0.4379.0. I've tried searching for:

- shared
- array
- buffer
- gate
- restrict
- coop
- coep

…but can't find anything relevant.

Reply:
I've just checked the CL: https://chromium-review.googlesource.com/c/chromium/src/+/2579721
This feature is in Canary starting 89.0.4351.0 - https://chromiumdash.appspot.com/commit/5089ddb0638765b738ce9690ee00be44d2e19a9c

But the feature is not UI enabled, it need's to be set via command line parameter.